plugins/source/aws/docs/tables/README.md

```diff
@@ -568,7 +568,11 @@
 - [aws_ssm_sessions](../../../../../website/tables/aws/aws_ssm_sessions.md)
 - [aws_ssoadmin_instances](../../../../../website/tables/aws/aws_ssoadmin_instances.md)
   - [aws_ssoadmin_permission_sets](../../../../../website/tables/aws/aws_ssoadmin_permission_sets.md)
+    - [aws_ssoadmin__customer_managed_policies](../../../../../website/tables/aws/aws_ssoadmin__customer_managed_policies.md)
     - [aws_ssoadmin_account_assignments](../../../../../website/tables/aws/aws_ssoadmin_account_assignments.md)
+    - [aws_ssoadmin_inline_policies](../../../../../website/tables/aws/aws_ssoadmin_inline_policies.md)
+    - [aws_ssoadmin_managed_policies](../../../../../website/tables/aws/aws_ssoadmin_managed_policies.md)
+    - [aws_ssoadmin_permission_boundaries](../../../../../website/tables/aws/aws_ssoadmin_permission_boundaries.md)
```

Comment:
I believe AWS uses Permissions Boundary with Permissions.

Let's keep that standard (permissions boundaries).  Checked as well our IAM Users and Roles.  We use permissions_boundary as the property.

Reply:
updated it to `aws_ssoadmin_permissions_boundaries`